test/e2e/soak_k8petstore.go

```diff
@@ -0,0 +1,231 @@
+/*
```

Comment:
The reason this is using an example as a mechanism to run a soak test.   Its not really a typcial examples.go test....  i can :
- rename the test file itself to something like `k8petstore.go` ? 
- or can move it if you'd move it into `examples.go`.  

I'd sort of lean to keep it separate but I realize this is a bit of a toss-up...